Dockerfile

```diff
@@ -24,21 +29,22 @@ ENV ZB_HOME=/usr/local/zeebe \
     ZEEBE_STANDALONE_GATEWAY=false
 ENV PATH "${ZB_HOME}/bin:${PATH}"
 
-RUN groupadd -g 1000 zeebe && \
-    adduser -u 1000 zeebe --system --ingroup zeebe
-
 WORKDIR ${ZB_HOME}
 EXPOSE 26500 26501 26502
 VOLUME ${ZB_HOME}/data
 
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
+RUN chmod g=u /etc/passwd
```

Comment:
I'm not a docker expert. I think earlier it was told that RUN should be merged together, but it seems that this is in the newest versions no longer the case, is this right? Related https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better

Reply:
You're right that it's usually better to merge them, in this case since those layers tend to be tiny and not updated often I didn't, but we could stay consistent. The line about it being less important in newer versions refers to the multi stage - since you don't care how many layers the non-target stages have, and having multiple run statements is usually more readable :man_shrugging: